docs/en/sql-reference/ansi.md

```diff
@@ -169,6 +179,7 @@ The following table lists cases when query feature works in ClickHouse, but beha
 | **F471**   | **Scalar subquery values**                                                                                               | **Yes**{.text-success}     |                                                                                                                                                                                           |
 | **F481**   | **Expanded NULL predicate**                                                                                              | **Yes**{.text-success}     |                                                                                                                                                                                           |
 | **F812**   | **Basic flagging**                                                                                                       | **No**{.text-danger}       |                                                                                                                                                                                           |
+| **S011**   | **Distinct data types** | | |
```

Comment:
Distinct **data types** - what's that?

Reply:
It is *user-defined types* (for example: https://docs.microsoft.com/en-us/openspecs/sql_standards/ms-tsqliso02/7962ac4f-e1c4-41c6-aa7c-6cc40f9fa7ce).